image/initrd/mkosi.conf

```diff
@@ -32,7 +33,6 @@ Packages=udev
          cryptsetup
 
 RemoveFiles=/var/log
-RemoveFiles=/var/cache
```

Comment:
> Changes required are keeping `/var/cache` around (which is reproducible for our images, so no problem), as mkosi needs files from it in the build process. 

What exactly is `/var/cache` being used for, and is it also needed for the initrd?

I'm also slightly worried about `/var/cache` becoming irreproducible in the future.

Reply:
Good point. During testing of the build, I've run into errors which seemed to be related to this, but verifying it now, it seems that removing them is not a problem. Updated the commit to leave the `RemoveFiles=` instructions as they were.